src/libs/ajax/WorkspaceDataService.ts

```diff
@@ -70,6 +85,30 @@ export const WorkspaceData = (signal) => ({
     );
     return res.json();
   },
+  getCapabilities: async (root: string): Promise<Capabilities> => {
+    return fetchWDS(root)('capabilities/v1', _.mergeAll([authOpts(), { signal, method: 'GET' }]))
+      .then(async (response) => {
+        const json = await response.json();
+        const capabilities: Capabilities = {};
+
+        for (const key in json) {
+          if (Object.prototype.hasOwnProperty.call(json, key)) {
+            // Default to false if not boolean
+            const jsonValue = typeof json[key] === 'boolean' ? json[key] : false;
+            capabilities[key as Capability] = jsonValue;
+          }
+        }
```

Comment:
This could possibly be done more concisely using [Lodash's mapValues](https://lodash.com/docs/4.17.15#mapValues).

```suggestion
        const capabilities: Capabilities = _.mapValues(val => value === true, json);
```

Reply:
I totally didn't see this suggestion the first time around and just went and tried to do this myself...tried yours and it works like a charm.  Thanks for providing the edit!

Fixed in d2d997bd3cfe979348464c464e2389d488620610